src/nouislider.js

```diff
@@ -170,7 +208,6 @@ function init(Survey) {
       }
     },
   };
-
   Survey.CustomWidgetCollection.Instance.addCustomWidget(widget, "customtype");
 }
 
```

Comment:
registerFunctionOnPropertiesValueChanged - we need to unregister this subscription